frontend/packages/jobs/src/functions/processSaveReview.ts

```diff
@@ -46,3 +85,49 @@ export const processSaveReview = async (
     throw error
   }
 }
+
+// Helper function to map category from review schema to CategoryEnum
+const mapCategoryEnum = (
+  category: string,
+):
+  | 'MIGRATION_SAFETY'
+  | 'DATA_INTEGRITY'
+  | 'PERFORMANCE_IMPACT'
+  | 'PROJECT_RULES_CONSISTENCY'
+  | 'SECURITY_OR_SCALABILITY' => {
+  const mapping: Record<
+    string,
+    | 'MIGRATION_SAFETY'
+    | 'DATA_INTEGRITY'
+    | 'PERFORMANCE_IMPACT'
+    | 'PROJECT_RULES_CONSISTENCY'
+    | 'SECURITY_OR_SCALABILITY'
+  > = {
+    'Migration Safety': 'MIGRATION_SAFETY',
+    'Data Integrity': 'DATA_INTEGRITY',
+    'Performance Impact': 'PERFORMANCE_IMPACT',
+    'Project Rules Consistency': 'PROJECT_RULES_CONSISTENCY',
+    'Security or Scalability': 'SECURITY_OR_SCALABILITY',
+  }
+  const result = mapping[category]
+  if (!result) {
+    throw new Error(`Invalid category: ${category}`)
+  }
+  return result
+}
+
+// Helper function to map severity from review schema to SeverityEnum
+const mapSeverityEnum = (
+  severity: string,
+): 'CRITICAL' | 'WARNING' | 'POSITIVE' => {
+  const mapping: Record<string, 'CRITICAL' | 'WARNING' | 'POSITIVE'> = {
+    high: 'CRITICAL',
+    medium: 'WARNING',
+    low: 'POSITIVE',
```

Comment:
Sorry! https://github.com/liam-hq/liam/pull/1079 is relevant.

Once #1079 is merged, this process may no longer be necessary. Please check. 🙏 @FunamaYukina

Reply:
Thank you! #1079 has been merged and I will modify the code!

Reply:
https://github.com/liam-hq/liam/pull/1067/commits/94260385c3ed6fc0403a5282944caf0897096931
I could remove `mapSeverityEnum` !